modules/rest-util/test/blaze/middleware/fhir/resource_test.clj

```diff
@@ -15,33 +16,44 @@
    [taoensso.timbre :as log])
   (:import
    [java.io ByteArrayInputStream]
-   [java.nio.charset StandardCharsets]))
+   [java.nio.charset StandardCharsets]
+   [java.util Base64]))
 
 (set! *warn-on-reflection* true)
 (st/instrument)
 (log/set-min-level! :trace)
 
 (test/use-fixtures :each tu/fixture)
 
-(defn wrap-error [handler]
+(defn- wrap-error [handler]
   (fn [request]
     (-> (handler request)
         (ac/exceptionally handler-util/error-response))))
 
-(def resource-handler
-  "A handler which just returns the :body from the request."
+(def ^:private resource-handler
+  "A handler which just returns the `:body` from a non-binary resource request."
```

Comment:
I don't use backticks for keywords. I think, I saw this in Clojure core docs. Keywords already stand out in text.